--- /usr/share/ibus-table/engine/table.py	2014-02-04 14:49:01.000000000 +0800
@@ -275,15 +275,15 @@
                 __lc = os.environ['LANG'].split('.')[0].lower()
 
             if __lc.find('_cn') != -1:
-                return 0
+                return 4
             # hk and tw is should use tc as default
             elif __lc.find('_hk') != -1 or __lc.find('_tw') != -1\
                     or __lc.find('_mo') != -1:
-                return 1
+                return 4
             else:
                 if self.db._is_chinese:
                     # if IME declare as Chinese IME
-                    return 0
+                    return 4
                 else:
                     return -1
         except:
